### DISCUSSION TITLE
Allow editing

### DISCUSSION THREAD

**Discussion by TarunavBA** [community/community#9762] [Other Feature Feedback, Questions, & Ideas] [answered]
Allow editing the file in codesearch like an online editor.

**Comment by look** [comment, marked as answer]
You can! Once in a search result type `.` to launch the [github.dev web editor](https://docs.github.com/en/codespaces/the-githubdev-web-based-editor).

You can also use `,` to quickly navigate to the GitHub.com view for the file, and use the single-file editor there (or launch github.dev with `.`).
